components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenuBuilder.kt

```diff
@@ -23,4 +24,24 @@ class BrowserMenuBuilder(
         val adapter = BrowserMenuAdapter(context, items)
         return BrowserMenu(adapter)
     }
+
+    /**
+     * Attaches the browser menu to the given menu button so that clicking on the button opens the menu.
+     */
+    fun attachTo(
+        menuButton: View,
+        orientation: BrowserMenu.Orientation = BrowserMenu.Orientation.DOWN,
+        onShow: () -> Unit = {},
+        onDismiss: () -> Unit = {}
+    ) {
+        menuButton.setOnClickListener {
+            build(menuButton.context).show(menuButton, orientation, endOfMenuAlwaysVisible, onDismiss)
```

Comment:
nit: this double `build` could probably go out of the listeners since they're the same in both.